recipes/kdalgorithms/all/conanfile.py

```diff
@@ -0,0 +1,52 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import get, copy, rmdir
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+import os
+
+required_conan_version = ">=2.0"
+
+class KDAlgorithmsConan(ConanFile):
+    name = "kdalgorithms"
+    license = "MIT"
+    description = "C++ Algorithm wrappers"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/KDAB/KDAlgorithms"
+    topics = ("c++14", "algorithmns", "kdab", "header-only")
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
```

Comment:
```suggestion
```

Avoid no_copy_source as we are running CMake and cache its files in the same source file.